docs/install/kubernetes.md

```diff
@@ -107,72 +122,70 @@ are being pulled locally for the first time (it will take an average of 10-15 mi
 your internet speed).
 ```
 
-To install the chart with the release name `mlrun-kit` use the following command.
+To install the chart with the release name `mlrun-ce` use the following command.
 Note the reference to the pre-created `registry-credentials` secret in `global.registry.secretName`:
 
 ```bash
 helm --namespace mlrun \
-    install mlrun-kit \
+    install mlrun-ce \
     --wait \
     --timeout 960s \
     --set global.registry.url=<registry-url> \
     --set global.registry.secretName=registry-credentials \
-    v3io-stable/mlrun-kit
+    --set global.externalHostAddress=<host-machine-address> \
+    mlrun-ce/mlrun-ce
 ```
 
-Where `<registry-url>` is the registry URL which can be authenticated by the `registry-credentials` secret (e.g., `index.docker.io/<your-username>` for Docker Hub).
+Where:
+ - `<registry-url>` is the registry URL which can be authenticated by the `registry-credentials` secret (e.g., `index.docker.io/<your-username>` for Docker Hub).
+ - `<host-machine-address>` is the IP address of the host machine (or `$(minikube ip)` if using minikube).
+
+Once the installation is complete, the helm command will print the URLs and Ports of all the MLRun CE services.
```

Comment:
Once the installation is complete, the helm command prints the URLs and Ports of all the MLRun CE services.